server/routes/confirmedOnReport/confirmedOnReport.ts

```diff
@@ -0,0 +1,38 @@
+import { Request, Response } from 'express'
+
+import ReportedAdjudicationsService from '../../services/reportedAdjudicationsService'
+import { formatName, formatTimestampToDate, formatTimestampToTime } from '../../utils/utils'
+
+export default class ConfirmedOnReportRoutes {
+  constructor(private readonly reportedAdjudicationsService: ReportedAdjudicationsService) {}
+
+  private renderView = async (req: Request, res: Response): Promise<void> => {
+    const { adjudicationNumber } = req.query
+    const { user } = res.locals
+
+    const adjudicationNumberValue: number = parseInt(adjudicationNumber as string, 10)
+    if (Number.isNaN(adjudicationNumberValue)) {
```

Comment:
Out of interest I guess this results in a 500? But I guess that's okay as we should always have an adjudication number. Do our users mess with urls?

Reply:
Actually that's a good point - it should be a 400. I will sort that out in a later PR